entity-framework/core/what-is-new/ef-core-9.0/whatsnew.md

```diff
@@ -425,6 +680,56 @@ The <xref:System.Linq.Enumerable.ToHashSet%2A?displayProperty=nameWithType> meth
 
 This enhancement was contributed by [@wertzui](https://github.com/wertzui). Many thanks!
 
+### Queries using Count != 0 are optimized
+
+> [!TIP]
+> The code shown here comes from [QuerySample.cs](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore9/QuerySample.cs).
+
+In EF8, the following LINQ query was translated to use the SQL `COUNT` function:
+
+<!--
+        var blogsWithPost = await context.Blogs
+            .Where(b => b.Posts.Count > 0)
+            .ToListAsync();
+-->
+[!code-csharp[NormalizeCount](../../../../samples/core/Miscellaneous/NewInEFCore9/QuerySample.cs?name=NormalizeCount)]
+
+EF9 now generates a more efficient translation using `EXISTS`:
```

Comment:
Show the previous SQL to show the optimization?

Reply:
😆 I actually deleted it because I thought, "Shay will be proud of me for keeping it shorter!"

Reply:
Will fix this in an update tomorrow. Want to get the docs live now.